database/indexPG.js

```diff
@@ -3,23 +3,23 @@ const knex = require('knex')({
   connection: {
     host: 'localhost',
     user: 'maikeru',
-    password: 'SDCpostgres',
+    password: 'postgres',
     database: 'item_summary',
   },
 });
 
 knex.schema.hasTable('items').then((exists) => {
```

Comment:
Curious where you defined your indexes for the schema?  You could do it here in this function.

Reply:
The auto-incremented id is indexed by default and the only index necessary for this application